xbmc/video/windows/GUIWindowVideoNav.cpp

```diff
@@ -859,6 +859,17 @@ void CGUIWindowVideoNav::GetContextButtons(int itemNumber, CContextButtons &butt
   }
 }
 
+bool CGUIWindowVideoNav::OnPopupMenu(int iItem)
+{
+  if (iItem >= 0 && iItem < m_vecItems->Size())
+  {
+    const auto item = m_vecItems->Get(iItem);
+    item->SetProperty("CheckAutoPlayNextItem", {true});
```

Comment:
Just curious why the `{true}`? String constructor or something similar?

Reply:
Oh well, this a mistake. Although it will not cause any problems, I will change that.